pkg/ocicni/ocicni.go

```diff
@@ -335,9 +342,15 @@ func (plugin *cniNetworkPlugin) syncNetworkConfig() error {
 
 	plugin.Lock()
 	defer plugin.Unlock()
-	if plugin.defaultNetName == "" {
+
+	// update default CNI network is defaultNetName is not specified during initialization
+	if plugin.isDefaultNetNameSpecified {
```

Comment:
@haircommander  Func `syncNetworkConfig` will be called when config files change, so `isDefaultNetNameSpecified` will be used not only once.